### PR TITLE
Misc. typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -I ../impacket-word-whitelist.txt` through downstream `curl`
word-whitelist:
```
ans
lsat
te
```

### CODE REVIEW

File: impacket/dns.py [hunk 1]
```diff
@@ -331,11 +331,11 @@ def parseCompressedMessage(self, buf, offset=0):
             offset += 1
             name = buf[offset:offset+byte]
             offset += byte
-            offset, unamed = self.parseCompressedMessage(buf, offset)
-            if not unamed:
```
Comment (luzpaz) [inline impacket/dns.py]: please review

File: impacket/dns.py [hunk 2]
```diff
@@ -331,11 +331,11 @@ def parseCompressedMessage(self, buf, offset=0):
...
+            offset, unnamed = self.parseCompressedMessage(buf, offset)
+            if not unnamed:
```
Comment (luzpaz) [inline impacket/dns.py]: please review

File: impacket/dns.py [hunk 3]
```diff
@@ -331,11 +331,11 @@ def parseCompressedMessage(self, buf, offset=0):
...
                 return (offset, name)
             else:
-                return (offset, name + "." + unamed)
+                return (offset, name + "." + unnamed)
```
Comment (luzpaz) [inline impacket/dns.py]: please review

Comment (asolino) [reply impacket/dns.py]: Looks OK!

File: impacket/krb5/kerberosv5.py [hunk 4]
```diff
@@ -112,7 +112,7 @@ def getKerberosTGT(clientName, password, domain, lmhash, nthash, aesKey='', kdcH
     reqBody['rtime'] = KerberosTime.to_asn1(now)
     reqBody['nonce'] =  random.getrandbits(31)
 
-    # Yes.. this shouldn't happend but it's inherited from the past
+    # Yes.. this shouldn't of happened but it's inherited from the past
```
Comment (asolino) [inline impacket/krb5/kerberosv5.py]: This one is wrong.. 'shouldn't happen' is the right phrase

Comment (asolino) [reply impacket/krb5/kerberosv5.py]: I'll fix this one myself don't worry.

**Comment by asolino** [comment]
Awesome @luzpaz, much needed :).. About to merge..

File: impacket/krb5/kerberosv5.py (hunk 4 above)
Comment (luzpaz) [reply impacket/krb5/kerberosv5.py]: Want me to fix this ?

**Comment by asolino** [comment]
Thanks a lot @luzpaz! Merged.

**Pull request merged**